README.md

```diff
@@ -10,6 +10,31 @@ Installation
 4. `sh ucsd-cs120-wi16-pa-tester/install.sh`
 5. `make test`
 
+## Usage
+#### Makefile
+the install.sh will echo the test target into the Makefile, 
+you can change the file name if necessary.
+
+2020-1-19: The mykernel2.c and mykernel2.h has been changed to mycode2.c and mycode2.h
+
+#### test.c
+the test.c contains the tests. you can modify it if you need.
+If you have logic in HandleTimerIntr, go to test.h and add a definition:
+
+`#define LOGIC_IN_HANDLETIMERINTR 1` 
+
+By default it is open.
+
+Also, for the project since it is running in single CPU, so whoever calls
+EndingProc should also be the one thats running in the CPU, which means the
+p given to the EndingProc should be the pid thats calling it.
+
+However, this does not apply to the tests, the p given to the
+EndingProc may not be the one thats calling it, so you should consider such
+situation in your code to utilize the test. But also, consider such situation, 
+where one process may kill another process, and you can see it is resonable to
+allow such operation in your projcet.
+
```

Comment:
To me, it seems like this is helping people "cheat" by giving them a hint on how to do the assignment. I vote that we take this section out of the README. After all, they should figure this out when they run the test ;)

Reply:
The reason why I put this into the README is that the test.c doesn`t really behave like the actual environment in the school`s server, as it only contains one CPU and only ends a process when the process itself calls Exit(). So in other words, the behavior that one process may kill another process is not tested by the autograder, and that`s why I think we can have an explanation here.

So, if you think that it`s giving hint to others it is totally fine to remove this part in README, or we can just replace this part with some simple notes here, like:
In this test program, there is such behavior that the running process may kill another process, which is not tested in the autograder.

I suggest we put the above simple notes here, because since it is not tested in the autograder, we don't need to add difficulty for students who are using the test program.

What do you think?

Reply:
I agree. I'm hesitant about the big blurb, but happy about the simple note :)